[2025-09-07T16:19:09.646+0000] {processor.py:161} INFO - Started process (PID=9188) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:19:09.646+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:19:09.649+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:19:09.648+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:19:09.809+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:19:09.807+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:19:09.809+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:19:09.822+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.183 seconds
[2025-09-07T16:19:40.013+0000] {processor.py:161} INFO - Started process (PID=9249) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:19:40.014+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:19:40.017+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:19:40.017+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:19:40.157+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:19:40.156+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:19:40.157+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:19:40.169+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.164 seconds
[2025-09-07T16:20:10.760+0000] {processor.py:161} INFO - Started process (PID=9310) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:20:10.761+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:20:10.763+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:20:10.763+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:20:10.890+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:20:10.889+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:20:10.890+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:20:10.899+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.146 seconds
[2025-09-07T16:20:41.342+0000] {processor.py:161} INFO - Started process (PID=9371) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:20:41.343+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:20:41.346+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:20:41.345+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:20:41.473+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:20:41.472+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:20:41.473+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:20:41.480+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.146 seconds
[2025-09-07T16:21:12.129+0000] {processor.py:161} INFO - Started process (PID=9432) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:21:12.130+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:21:12.132+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:21:12.132+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:21:12.274+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:21:12.273+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:21:12.274+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:21:12.284+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.161 seconds
[2025-09-07T16:21:42.862+0000] {processor.py:161} INFO - Started process (PID=9493) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:21:42.863+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:21:42.865+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:21:42.865+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:21:43.003+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:21:43.002+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:21:43.003+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:21:43.013+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.156 seconds
[2025-09-07T16:22:13.525+0000] {processor.py:161} INFO - Started process (PID=9554) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:13.526+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:22:13.529+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:22:13.529+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:13.653+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:22:13.652+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:22:13.653+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:13.661+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.143 seconds
[2025-09-07T16:22:25.745+0000] {processor.py:161} INFO - Started process (PID=9562) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:25.747+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:22:25.751+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:22:25.750+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:25.969+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:22:25.967+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:22:25.969+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:25.980+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.248 seconds
[2025-09-07T16:22:56.526+0000] {processor.py:161} INFO - Started process (PID=9623) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:56.527+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:22:56.529+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:22:56.528+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:56.668+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:22:56.667+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:22:56.668+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:22:56.675+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.157 seconds
[2025-09-07T16:23:27.125+0000] {processor.py:161} INFO - Started process (PID=9684) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:23:27.126+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:23:27.129+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:23:27.129+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:23:27.257+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:23:27.256+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:23:27.257+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:23:27.264+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.145 seconds
[2025-09-07T16:23:57.744+0000] {processor.py:161} INFO - Started process (PID=9745) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:23:57.747+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:23:57.752+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:23:57.752+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:23:57.924+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:23:57.923+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:23:57.924+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:23:57.933+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.197 seconds
[2025-09-07T16:24:28.550+0000] {processor.py:161} INFO - Started process (PID=9806) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:24:28.551+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:24:28.554+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:24:28.553+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:24:28.702+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:24:28.701+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:24:28.702+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:24:28.710+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.169 seconds
[2025-09-07T16:24:59.069+0000] {processor.py:161} INFO - Started process (PID=9867) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:24:59.070+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:24:59.071+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:24:59.071+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:24:59.196+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:24:59.193+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:24:59.196+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:24:59.204+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.142 seconds
[2025-09-07T16:25:29.574+0000] {processor.py:161} INFO - Started process (PID=9928) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:25:29.575+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:25:29.579+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:25:29.578+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:25:29.728+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:25:29.727+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:25:29.728+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:25:29.736+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.171 seconds
[2025-09-07T16:26:00.137+0000] {processor.py:161} INFO - Started process (PID=9989) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:00.140+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:26:00.144+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:26:00.143+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:00.324+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:26:00.323+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:26:00.324+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:00.333+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.205 seconds
[2025-09-07T16:26:04.208+0000] {processor.py:161} INFO - Started process (PID=9991) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:04.209+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:26:04.211+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:26:04.210+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:04.332+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:26:04.331+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:26:04.332+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:04.339+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.137 seconds
[2025-09-07T16:26:34.852+0000] {processor.py:161} INFO - Started process (PID=10052) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:34.854+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:26:34.857+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:26:34.857+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:35.010+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:26:35.009+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:26:35.010+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:35.018+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.175 seconds
[2025-09-07T16:26:49.458+0000] {processor.py:161} INFO - Started process (PID=10107) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:49.459+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:26:49.460+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:26:49.460+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:49.655+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:26:49.654+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:26:49.655+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:26:49.661+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.207 seconds
[2025-09-07T16:27:20.115+0000] {processor.py:161} INFO - Started process (PID=10168) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:27:20.117+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:27:20.118+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:27:20.118+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:27:20.417+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:27:20.416+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:27:20.417+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:27:20.425+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.317 seconds
[2025-09-07T16:27:29.302+0000] {processor.py:161} INFO - Started process (PID=10176) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:27:29.303+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:27:29.305+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:27:29.305+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:27:29.494+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:27:29.493+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:27:29.494+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:27:29.501+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.203 seconds
[2025-09-07T16:28:00.099+0000] {processor.py:161} INFO - Started process (PID=10237) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:00.101+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:28:00.103+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:28:00.102+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:00.258+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:28:00.257+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:28:00.258+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:00.267+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.173 seconds
[2025-09-07T16:28:30.635+0000] {processor.py:161} INFO - Started process (PID=10298) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:30.636+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:28:30.637+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:28:30.637+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:30.825+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:28:30.824+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:28:30.825+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:30.830+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.201 seconds
[2025-09-07T16:28:42.800+0000] {processor.py:161} INFO - Started process (PID=10333) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:42.801+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:28:42.803+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:28:42.803+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:43.090+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:28:43.088+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:28:43.091+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:28:43.103+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.308 seconds
[2025-09-07T16:29:13.531+0000] {processor.py:161} INFO - Started process (PID=10394) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:29:13.532+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:29:13.534+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:29:13.533+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:29:13.852+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:29:13.851+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:29:13.852+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:29:13.858+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.333 seconds
[2025-09-07T16:29:44.289+0000] {processor.py:161} INFO - Started process (PID=10455) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:29:44.289+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:29:44.291+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:29:44.291+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:29:44.509+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:29:44.506+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:29:44.510+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:29:44.518+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.235 seconds
[2025-09-07T16:30:15.007+0000] {processor.py:161} INFO - Started process (PID=10516) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:30:15.009+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:30:15.010+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:30:15.010+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:30:15.400+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:30:15.397+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:30:15.401+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:30:15.411+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.407 seconds
[2025-09-07T16:30:45.824+0000] {processor.py:161} INFO - Started process (PID=10577) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:30:45.825+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:30:45.827+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:30:45.827+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:30:46.231+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:30:46.229+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:30:46.231+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:30:46.242+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.424 seconds
[2025-09-07T16:31:16.705+0000] {processor.py:161} INFO - Started process (PID=10638) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:31:16.706+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:31:16.708+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:31:16.707+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:31:17.051+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:31:17.049+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:31:17.051+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:31:17.057+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.358 seconds
[2025-09-07T16:31:47.465+0000] {processor.py:161} INFO - Started process (PID=10699) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:31:47.465+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:31:47.467+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:31:47.466+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:31:47.792+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:31:47.790+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:31:47.792+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:31:47.800+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.341 seconds
[2025-09-07T16:32:18.260+0000] {processor.py:161} INFO - Started process (PID=10760) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:32:18.262+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:32:18.264+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:32:18.264+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:32:18.602+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:32:18.599+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:32:18.602+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:32:18.613+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.358 seconds
[2025-09-07T16:32:49.093+0000] {processor.py:161} INFO - Started process (PID=10821) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:32:49.093+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:32:49.095+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:32:49.095+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:32:49.377+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:32:49.376+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:32:49.378+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:32:49.399+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.311 seconds
[2025-09-07T16:33:19.912+0000] {processor.py:161} INFO - Started process (PID=10882) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:33:19.913+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:33:19.915+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:33:19.914+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:33:20.280+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:33:20.279+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:33:20.281+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:33:20.292+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.386 seconds
[2025-09-07T16:33:51.019+0000] {processor.py:161} INFO - Started process (PID=10943) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:33:51.019+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:33:51.020+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:33:51.020+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:33:51.422+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:33:51.418+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:33:51.423+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:33:51.437+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.421 seconds
[2025-09-07T16:34:21.844+0000] {processor.py:161} INFO - Started process (PID=11004) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:34:21.845+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:34:21.846+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:34:21.846+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:34:22.183+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:34:22.182+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:34:22.183+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:34:22.188+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.351 seconds
[2025-09-07T16:34:52.594+0000] {processor.py:161} INFO - Started process (PID=11065) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:34:52.595+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:34:52.598+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:34:52.598+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:34:52.825+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:34:52.824+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:34:52.825+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:34:52.830+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.244 seconds
[2025-09-07T16:35:23.221+0000] {processor.py:161} INFO - Started process (PID=11126) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:23.222+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:35:23.224+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:35:23.223+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:23.502+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:35:23.499+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:35:23.502+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:23.508+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.293 seconds
[2025-09-07T16:35:54.017+0000] {processor.py:161} INFO - Started process (PID=11193) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:54.018+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:35:54.021+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:35:54.021+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:54.320+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:35:54.319+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:35:54.320+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:54.328+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.323 seconds
[2025-09-07T16:35:54.594+0000] {processor.py:161} INFO - Started process (PID=11214) to work on /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:54.596+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/yfinance_csvimport.py for tasks to queue
[2025-09-07T16:35:54.597+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:35:54.597+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:54.810+0000] {logging_mixin.py:188} INFO - [2025-09-07T16:35:54.810+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/yfinance_csvimport.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/yfinance_csvimport.py", line 4, in <module>
    import yfinance as yf
ModuleNotFoundError: No module named 'yfinance'
[2025-09-07T16:35:54.811+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/yfinance_csvimport.py
[2025-09-07T16:35:54.816+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/yfinance_csvimport.py took 0.225 seconds
